packages/trader/src/AppV2/Components/ActiveSymbolsList/active-symbols-list.tsx

```diff
@@ -0,0 +1,74 @@
+import { ActionSheet, Tab } from '@deriv-com/quill-ui';
+import { observer } from '@deriv/stores';
+import React, { useState, useEffect, useRef } from 'react';
+import SymbolsSearchField from '../SymbolsSearchField';
+import MarketCategories from '../MarketCategories';
+import useActiveSymbols from 'AppV2/Hooks/useActiveSymbols';
+import SymbolSearchResults from '../SymbolSearchResults';
+import { useTraderStore } from 'Stores/useTraderStores';
+
+type TActiveSymbolsList = {
+    isOpen: boolean;
+    setIsOpen: (input: boolean) => void;
+};
+
+const ActiveSymbolsList = observer(({ isOpen, setIsOpen }: TActiveSymbolsList) => {
+    const { default_symbol } = useActiveSymbols({});
+
+    const [isSearching, setIsSearching] = useState(false);
+    const [selectedSymbol, setSelectedSymbol] = useState(default_symbol);
+    const [searchValue, setSearchValue] = useState('');
+    const { symbol } = useTraderStore();
+
+    const marketCategoriesRef = useRef<HTMLDivElement | null>(null);
```

Comment:
```suggestion
    const marketCategoriesRef = useRef<HTMLDivElement>(null);
```
should infer the `null` type automatically :)